db/schema.rb

```diff
@@ -35,6 +35,23 @@
     t.boolean  "is_featured"
   end
 
+  create_table "installs", force: :cascade do |t|
```

Comment:
This got back in here because I don't think you pulled from master before you made this branch.  Can you try to pull latest from master, then rebase this branch from master, then re-push to update this PR?

Here's how to do that if you need help:
git checkout master
git pull
git checkout update_location_pages
git rebase master
git push origin update_location_pages --force